samples/testing-frameworks/appium/client-side/java/src/test/java/com/testdroid/appium/ios_biometrics/IosBiometryAppiumExampleTest.java

```diff
@@ -0,0 +1,46 @@
+package com.testdroid.appium.ios_biometrics;
+
+import com.google.common.collect.ImmutableMap;
+import com.testdroid.appium.BaseIOSTest;
+
+import io.appium.java_client.ios.IOSDriver;
+import org.openqa.selenium.Alert;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.concurrent.TimeUnit;
+
+class IosBiometryAppiumExampleTest extends BaseIOSTest {
+
+
+
+    @BeforeEach
+    public void setUp() throws Exception {
+        setUpTest();
+    }
+
+    @AfterEach
+    public void tearDown() {
+        quitAppiumSession();
+    }
+
+    @Test
+    @Deprecated
+    public void mainPageTest() {
+        wd.manage().timeouts().implicitlyWait(60, TimeUnit.SECONDS);
+        takeScreenshot("start_screen");
+        wd.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+        wd.executeScript("mobile: alert", ImmutableMap.of("action", "accept", "buttonLabel", "OK"));
+        wd.findElement(By.xpath("(//*[@name='Biometric authentication'])[3]")).click();
+        wd.findElement(By.xpath("//XCUIElementTypeOther[@name='Ask biometric authentication']")).click();
+        takeScreenshot("Ask_Biometrics");
+        wd.executeScript("mobile: alert", ImmutableMap.of("action", "accept", "buttonLabel", "Pass"));
+        takeScreenshot("PassBiometrics");
+    }
```

Comment:
You did a step in android test that checks failing of biometric authentication. Maybe let's add it to iOS test too?